mubench.pipeline/tests/data/stub_detector.py

```diff
@@ -3,4 +3,5 @@
 
 class StubDetector(Detector):
     def __init__(self):
-        super().__init__("-detectors-", "StubDetector", [], cli_version=RunnerInterfaceTestImpl.TEST_VERSION)
+        self._get_release_info = lambda *_: {'tag': 'latest', 'md5': None, 'cli_version': RunnerInterfaceTestImpl.TEST_VERSION}
```

Comment:
Do we need this hack to set the values? Since the releases file does not exist in the tests, the detector should use the defaults. This is fine for `tag` and `md5`, and we can set `self.cli_version = RunnerInterfaceTestImpl.TEST_VERSION` after calling the super constructor, can't we?